lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
+  final String hashtag;
+  final Color color;
+  final int size;
+  final bool rotated;
+}
+
+class FlutterColors {
+  const FlutterColors._();
+
+  static const Color yellow = Color(0xFFFFC108);
+
+  static const Color white = Color(0xFFFFFFFF);
+
+  static const Color blue400 = Color(0xFF13B9FD);
+  static const Color blue600 = Color(0xFF0175C2);
+  static const Color blue = Color(0xFF02569B);
+
+  static const Color gray100 = Color(0xFFD5D7DA);
+  static const Color gray600 = Color(0xFF60646B);
+  static const Color gray = Color(0xFF202124);
+}
+
+const List<FlutterHashtag> kFlutterHashtags = <FlutterHashtag>[
```

Comment:
This should be moved to the `mock` folder. Should also rename the var to `mockKeywords`

Reply:
Addressed in commit e7631b35d688c4435efa2dfee59910f562013293